### ISSUE TITLE
Duplicate IDs for search form in Knowledge Base

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#11072]
## Description
In the Knowledge Base, there is a second search form in the left nav sidebar below the navigation.  This is hidden with CSS so it cannot be seen or accessed, however it's still in the DOM and therefore throwing an accessibility error because it has the same IDs as the visible search form in the header.

Since this form is hidden anyway, is this needed? Can we remove it and clear the axe error? If it is needed, can we change the IDs to make each search field unique?

Hidden:
![image](https://user-images.githubusercontent.com/106678594/194075096-9bda6007-68ea-4662-bc19-bab58dcca2a2.png)
Not hidden:
![image](https://user-images.githubusercontent.com/106678594/194075192-75a22349-d21b-407a-86ed-633b241a96c5.png)

## Acceptance Criteria
- [ ] Remove the hidden search form in the sidebar
- [ ] Accessibility Lead review

**Comment by omahane** [comment]
TODO:
Remove `block.block.vagovclaro_knowledge_base_search_form` from config sync.

**Comment by EWashb** [comment]
Do we anticipate that this will be complete this sprint @laflannery and @omahane?

**Comment by omahane** [comment]
For sure.
